src/devices/Amg88xx/samples/Amg88xx.Sample.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Threading;
+using UnitsNet;
+
+namespace Iot.Device.Amg88xx.Samples
+{
+    /// <summary>
+    /// Samples for using the Amg88xx binding incl. interrupt handling.
+    /// Refer to the README.md for the required hardware setup to run this sample.
+    /// </summary>
+    public class Program
+    {
+        /// <summary>
+        /// Main entry point
+        /// </summary>
+        public static void Main(string[] args)
+        {
+            // Setup I2C bus for communicating with the AMG8833 sensor.
+            // Note: if you're using a breakout board check which address is configured by the logic level
+            // of the sensor's AD_SELECT pin.
+            const int I2cBus = 1;
+            I2cConnectionSettings i2cSettings = new I2cConnectionSettings(I2cBus, Amg88xx.AlternativeDeviceAddress);
+            I2cDevice i2cDevice = I2cDevice.Create(i2cSettings);
+
+            // Setup GPIO controller for receiving interrupts from the sensor's INT pin (pin 5)
+            // and for driving an LED (pin 6) as an interrupt indicator.
+            GpioController ioController = new GpioController();
```

Comment:
Would it makes sense to integrate directly the interupt mechanism in the device itself? if no pin are passed or negative, then no interrupt, if some are passed, then interupt mode can be managed. something that would make it even more integrated and transparent